src/main/java/entity/PlayerStatistic.java

```diff
@@ -0,0 +1,254 @@
+package entity;
+
+public class PlayerStatistic {
+    private final int age;
+    private final int assists;
```

Comment:
All fields were added as final. This is a good practice to enforce immutability.